include/coreutil.h

```diff
@@ -393,6 +478,7 @@ inline void XXHash_256(const void* input, size_t len, uint64_t* out)
     out[3] += v4;
 }
 
+//! Utility class easing the process of finding memory leaks. Usable only in debug build. Thread-safe.
```

Comment:
Doesn't have a Windows implementation yet.

Reply:
It's mostry not documented. I'll have to ask what `getBackTrace` actually do because i couldn't figure out.